### PR TITLE
demo pr

### PR DESCRIPTION
**Pull request by slicksammy**
this pr updates the README

### CODE REVIEW

**Comment by slicksammy** [comment]
test comment

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # README
 
+-- first pr
```
Comment (slicksammy) [inline README.md:3]: review comment

Comment (slicksammy) [reply README.md:3]: another test

Comment (slicksammy) [reply README.md:3]: i like this

Comment (slicksammy) [reply README.md:3]: very good

Comment (slicksammy) [reply README.md:3]: another comment

Comment (slicksammy) [reply README.md:3]: i like it

Comment (slicksammy) [reply README.md:3]: what is this? @pear-on can you help?

Comment (slicksammy) [inline README.md:3]: new thread

Comment (slicksammy) [reply README.md:3]: test comment

Comment (slicksammy) [reply README.md:3]: trying to test

Comment (slicksammy) [reply README.md:3]: comment

Comment (slicksammy) [reply README.md:3]: testing

Comment (slicksammy) [reply README.md:3]: what is this?

Comment (slicksammy) [reply README.md:3]: what is this?

Comment (slicksammy) [reply README.md:3]: what is this?

Comment (pear-on) [reply README.md:3]: Hello, this is a first pull request made to demo the changes done to the README file.

Comment (pear-on) [reply README.md:3]: Thank you for your comment. Your pull request is being reviewed.

Comment (pear-on) [reply README.md:3]: Thank you for submitting your comment and pull request. We're currently reviewing it and will get back to you shortly.

Comment (slicksammy) [reply README.md:3]: @pear-on explain this diff

Comment (slicksammy) [reply README.md:3]: @pear-on explain this diff

Comment (slicksammy) [reply README.md:3]: @pear-on explain this diff

Comment (slicksammy) [reply README.md:3]: @pear-on explain this diff

Comment (pear-on) [reply README.md:3]: Hello, this diff is showing the differences between the original file and the updated file. The '+-- first pr' indicates a line that has been added. In this case, '-- first pr' has been added to the README file.

Comment (slicksammy) [reply README.md:3]: test

Comment (slicksammy) [reply README.md:3]: @pear-on please explain this diff

Comment (pear-on) [reply README.md:3]: Hello, this diff shows that two new lines have been added to the README file, specifically '-- first pr'. This probably signifies the first change made to the project via a pull request.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+
 This README would normally document whatever steps are necessary to get the
 application up and running.
```
Comment (slicksammy) [inline README.md:7]: @pear-on what is this?

Comment (pear-on) [reply README.md:7]: Hi there! The added lines in the README are for documentation purposes. They acknowledge the first pull request made on this repository. If you have further questions, feel free to ask.

Comment (slicksammy) [reply README.md:7]: @pear-on explain this diff

Comment (pear-on) [reply README.md:7]: Hello, the diff you're referring to shows the addition of two new lines in the README file with the text `-- first pr`. This change was made as part of the pull request. It's always a good practice to document any significant changes made in the commit within the README file. Hope this clears your doubt.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
...
```
Comment (slicksammy) [inline README.md:6]: @pear-on explain this diff

Comment (pear-on) [reply README.md:6]: The diff represents a change in the README file of this repository. Two lines have been added (`+-- first pr
+
`). This indicates that a description or note about the first PR has been added to the README. Let me know if you need further explanation.

**Pull request closed without merging**